etl/steps/data/garden/artificial_intelligence/2024-07-16/cset.meta.yml

```diff
@@ -0,0 +1,232 @@
+
+  description_key_patents: &description_key_patents |-
+    World aggregate does not include data for Micronesia, Tonga, Samoa, Kiribati, Fiji, Papua New Guinea, Palau, Tuvalu, Bermuda, Armenia, Belarus, Georgia, Gibraltar, Jersey, Kosovo, Moldova, Isle of Man, Iceland, Albania, Andorra, Bosnia and Herzegovina, Malta, Montenegro, San Marino, North Macedonia, Liechtenstein, Monaco, Vatican City, Guernsey, Afghanistan, Kyrgyzstan, Bahrain, Laos, Bangladesh, Lebanon, Bhutan, Maldives, Cambodia, Syria, Tajikistan, Cyprus, Mongolia, North Korea, Myanmar, Timor-Leste, Nepal, Turkmenistan, Pakistan, Palestine, Iraq, United Arab Emirates, Uzbekistan, Kazakhstan, Qatar, Vietnam, Yemen, Kuwait, Algeria, Cape Verde, Equatorial Guinea, Swaziland, Namibia, Central African Republic (the), Angola, Ethiopia, Niger, Benin, Gabon, Nigeria, Botswana, Gambia, Rwanda, Burkina Faso, Ghana, São Tomé and Príncipe, Burundi, Guinea, Senegal, Guinea-Bissau, Seychelles, Cameroon, Sierra Leone, Lesotho, Somalia, Chad, Liberia, Libya, South Sudan, Congo, Madagascar, Sudan, Côte d'Ivoire, Malawi, Togo, Mali, Djibouti, Mauritania, Uganda, Egypt, Mauritius, Tanzania, Zambia, Eritrea, Mozambique, Zimbabwe, Comoros, Antigua and Barbuda, Bolivia, Suriname, Nicaragua, Dominican Republic, Bahamas, Ecuador, Paraguay, Barbados, Saint Vincent and the Grenadines, El Salvador, Belize, Grenada, Saint Kitts and Nevis, Guatemala, Guyana, Haiti, Honduras, Trinidad and Tobago, Jamaica, Venezuela, Puerto Rico, Cayman Islands (the), Turks and Caicos Islands, Saint Lucia, and Dominica.
+
+  description_key_investment: &description_key_investment |-
+    World aggregate does not include data for Micronesia, Tonga, Samoa, Kiribati, Fiji, Papua New Guinea, Palau, Tuvalu, Gibraltar, Jersey, Kosovo, Moldova, Isle of Man, Andorra, Montenegro, San Marino, Liechtenstein, Monaco, Vatican City, Afghanistan, Kyrgyzstan, Laos, Hong Kong, Bhutan, Brunei Darussalam, Maldives, Syria, North Korea, Myanmar, Timor-Leste, Nepal, Turkmenistan, Palestine, Yemen, Kuwait, Cape Verde, Equatorial Guinea, Swaziland, Namibia, Central African Republic (the), Angola, Ethiopia, Niger, Benin, Gabon, Gambia, Rwanda, Burkina Faso, São Tomé and Príncipe, Burundi, Guinea, Guinea-Bissau, Cameroon, Sierra Leone, Lesotho, Somalia, Chad, Liberia, Libya, South Sudan, Congo, Sudan, Malawi, Togo, Mali, Djibouti, Mauritania, Eritrea, Mozambique, Comoros, Antigua and Barbuda, Bolivia, Suriname, Nicaragua, Bahamas, Saint Vincent and the Grenadines, Grenada, Guyana, Haiti, Honduras, Cuba, Turks and Caicos Islands, Saint Lucia, and Dominica.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  cset:
+    variables:
+      disclosed_investment:
+        title: Disclosed investment
+        unit: US dollars
```

Comment:
It sounds like the adjusting for inflation situation is pretty complicated, but not sure if it makes sense to have something here in line with the [harmonizing document?](https://docs.google.com/document/d/1tcknTryJAxOR0yv8x38s3jkONq-TY3T4HNCEcZQ_c-s/edit?pli=1#heading=h.qd6mxo3fpwgz) e.g. constant 2021 US$